logger/logger.go

```diff
@@ -95,7 +96,27 @@ func SetLoggingLevel(level string) {
 	if err != nil {
 		panic(err)
 	}
-	conf.level.SetLevel(zapLevel)
+	Conf.level.SetLevel(zapLevel)
+}
+
+// This exists to be able to export the logging level configs to the debugserver
+func (loggingConf DynamicLoggingConfig) SetMinLevel(level lager.LogLevel) {
+	Conf.level.SetLevel(toZapLevel(level))
+}
+
+func toZapLevel(level lager.LogLevel) zapcore.Level {
+	switch level {
+	case lager.DEBUG:
+		return zapcore.DebugLevel
+	case lager.INFO:
+		return zapcore.InfoLevel
```

Comment:
*Small suggestion*

If `zapcore.InfoLevel` is the default case, I don't think you need to add a specific case for `lager.INFO` unless there are explicit reasons for doing so.

Reply:
I thought about it, and chose to go explicit over implicit in the handling of the lager libraries while also being verbose about the log leveling hierarchy and translations. There are only a 4 cases with lager to the 6 with zapcore. The lager interface is the one we need to satisfy, so handling each was the priority in my mind.

["Explicit is better than implicit"](https://peps.python.org/pep-0020/) mentality.